maintainers/maintainer-list.nix

```diff
@@ -10868,6 +10868,16 @@
     githubId = 1607770;
     name = "Ulrik Strid";
   };
+  unclechu = {
```

Comment:
Please add yourself as a maintainer by adding an alphabetically sorted entry to `maintainers/maintainer-list.nix` and commit this change into a separate commit with the message ``maintainers: add NAME`` into this PR.

Reply:
@SuperSandro2000 to clarify, it is alphabetically sorted on the top-level. It goes after `ulrikstrid`.

To make sure I don’t mess up I put my entry after last entry that start with `u`, selected all entries that are starting with `u` and applied this command in vim:

``` viml
'<,'>!perl -E '@o=(); while (<>) { if (/^  };/) { push @o, $x.$_; $x=""; } else { $x.=$_ } }; print join "", sort @o' 2>&1
```

Reply:
@SuperSandro2000 I force-pushed a change where I split the add to the maintainers list into a separate commit.

Reply:
That is a standard reply text I select when there is something about the maintainer entry not being in an extra commit or not being sorted. It was actually sorted in your case :)